templates/search.hbs

```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+	<script src="/static/js/main.js"></script>
```

Comment:
Could you move these to the bottom of the body, where the rest of the scripts are?